fastlane/Fastfile

```diff
@@ -0,0 +1,42 @@
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Description of what the lane does"
+  lane :custom_lane do
+    # add actions here: https://docs.fastlane.tools/actions
+  end
+
+  lane :certificates do
+    # development
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.FeedbackSample",  git_branch: "foresee", type: "development", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.LocalNotificationSample",  git_branch: "foresee", type: "development", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.BasicSample",  git_branch: "foresee", type: "development", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.BasicSwiftSample",  git_branch: "foresee", type: "development", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.CustomInvitationSample",  git_branch: "foresee", type: "development", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.ComplexSample",  git_branch: "foresee", type: "development", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.ContactInvitationSample",  git_branch: "foresee", type: "development", readonly: true)
+    # adhoc
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.FeedbackSample",  git_branch: "foresee", type: "adhoc", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.LocalNotificationSample",  git_branch: "foresee", type: "adhoc", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.BasicSample",  git_branch: "foresee", type: "adhoc", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.BasicSwiftSample",  git_branch: "foresee", type: "adhoc", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.CustomInvitationSample",  git_branch: "foresee", type: "adhoc", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.ComplexSample",  git_branch: "foresee", type: "adhoc", readonly: true)
+    match(git_url: "git@github.com:foreseecode/mobile-fastlane-match.git", app_identifier: "com.foresee.samples.ContactInvitationSample",  git_branch: "foresee", type: "adhoc", readonly: true)
```

Comment:
I think you could simplify this by doing this once for each certificate type, e.g.:
```
    match(type: "development")
    match(type: "adhoc")
```
Your Matchfile should supply the rest of the arguments, and includes all the app_identifiers 🙂